ext/opentelemetry-ext-jinja2/tests/test_jinja2.py

```diff
@@ -0,0 +1,149 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+import jinja2
+
+from opentelemetry import trace as trace_api
+from opentelemetry.ext.jinja2 import Jinja2Instrumentor
+from opentelemetry.test.test_base import TestBase
+from opentelemetry.trace import get_tracer
+
+TEST_DIR = os.path.dirname(os.path.realpath(__file__))
+TMPL_DIR = os.path.join(TEST_DIR, "templates")
+
+
+class TestJinja2Instrumentor(TestBase):
+    def setUp(self):
+        super().setUp()
+        self.instrumentor = Jinja2Instrumentor()
```

Comment:
Nit: `Jinja2Instrumentor` is a singleton, so you don't need to save the instance.